src/components/Modal/index.js

```diff
@@ -68,6 +71,20 @@ export function Modal({
                   style={closeButtonStyle}
                   className={`pointer-events-none sm:pointer-events-auto absolute top-0 right-0 ${closeButtonClass}`}
                 >
+                  {dropDownVisible && (
+                    <TopologyDropdownMenu
```

Comment:
@OleksandrSekker look inside TopologyDropdownMenu it's not reusable component anymore

Reply:
removed